package.json

```diff
@@ -43,12 +43,12 @@
 		"uuid": "^9.0.0"
 	},
 	"peerDependencies": {
+		"@rollup/plugin-json": "^6.0.0",
```

Comment:
this plugin isn't needed anymore since react-syntax-highlighter isn't being compiled into the package anymore